common/src/model_deployment/deploy.py

```diff
@@ -0,0 +1,243 @@
+    )
+    parser.add_argument(
+        "--instance_count",
+        type=int,
+        help="Number of compute instances to deploy model",
+        default=1,
+        choices=range(1, 20),
+    )
+    parser.add_argument(
+        "--max_concurrent_requests_per_instance",
+        type=int,
+        default=1,
+        help="Maximum concurrent requests to be handled per instance",
+    )
+    parser.add_argument(
+        "--request_timeout_ms",
+        type=int,
+        default=5000,
+        help="Request timeout in ms. Max limit is 90000.",
+    )
+    parser.add_argument(
+        "--max_queue_wait_ms",
+        type=int,
+        default=500,
+        help="Maximum queue wait time of a request in ms",
+    )
+    parser.add_argument(
+        "--failure_threshold_readiness_probe",
+        type=int,
+        default=10,
+        help="No of times system will try after failing the readiness probe",
+        choices = range(1,50)
+    )
+    parser.add_argument(
+        "--success_threshold_readiness_probe",
+        type=int,
+        default=1,
+        help="The minimum consecutive successes for the readiness probe to be considered successful after having failed",
+        choices = range(1,50)
+    )
+    parser.add_argument(
+        "--timeout_readiness_probe",
+        type=int,
+        default=10,
+        help="The number of seconds after which the readiness probe times out",
+        choices = range(1,500)
+    )
+    parser.add_argument(
+        "--period_readiness_probe",
+        type=int,
+        default=10,
+        help="How often (in seconds) to perform the readiness probe",
+        choices = range(1,500)
+    )
+    parser.add_argument(
+        "--initial_delay_readiness_probe",
+        type=int,
+        default=10,
+        help="The number of seconds after the container has started before the readiness probe is initiated",
+        choices = range(1,500)
+    )
+    parser.add_argument(
+        "--failure_threshold_liveness_probe",
+        type=int,
+        default=30,
+        help="No of times system will try after failing the liveness probe",
+        choices = range(1,50)
+    )
+    parser.add_argument(
+        "--timeout_liveness_probe",
+        type=int,
+        default=10,
+        help="The number of seconds after which the liveness probe times out",
+        choices = range(1,500)
+    )
+    parser.add_argument(
+        "--period_liveness_probe",
+        type=int,
+        default=10,
+        help="How often (in seconds) to perform the liveness probe",
+        choices = range(1,500)
+    )
+    parser.add_argument(
+        "--initial_delay_liveness_probe",
+        type=int,
+        default=10,
+        help="The number of seconds after the container has started before the liveness probe is initiated",
+        choices = range(1,500)
+    )
+    parser.add_argument(
+        "--egress_public_network_access",
+        type=int,
+        default="enabled",
```

Comment:
type is `int` here. But default provided is a string. Please change type to string

Reply:
Thanks Ayush